content/source/docs/enterprise/workspaces/run-ui.html.md

```diff
@@ -25,6 +25,8 @@ If the code in version control hasn't changed but you've modified some variables
 
 Manually queueing a plan requires write or admin access.
 
+If the workspace has a plan that is still in the [Plan stage](./run-basics.html#2-the-plan-stage) when a new plan is queued, you can either wait for it to complete, or visit the "Current Run" page and click "Run this plan now". Be aware that this will terminate the current plan and unlock the workspace, which can lead to anomalies in behavior, but can be useful if the plans are long-running and the current plan is known not to have all the desired changes.
```

Comment:
As a user, I would like a little bit more info about these "anomalies!" Can this corrupt anything in the workspace, like the run history or anything? Does it make the UI go wonky? What am I getting into, here?

Reply:
I don't actually know - the text specifically says something about race conditions, the nature of which means unpredictable things can happen. I could mirror the same text here, I guess?

Reply:
Hmm. I'd at least like to know what the worst-case scenario is. Wonder who would know that? @ryanuber, maybe?

Reply:
I agree it would be ideal, but I'd also really like to get this out there, since stuck runs are kind of killing the support team right now and to the extent customers can self-resolve, it would help a lot...any thoughts?